### PR TITLE
Fixed reference to 'disabled' and removed ref from checkbox

### PR DESCRIPTION
**Pull request by JakeDawkins**
for https://github.com/NewSpring/Holtzmann/issues/1134

### CODE REVIEW

**Comment by JakeDawkins** [comment]
@johnthepink was there a reason there was a ref on the input for this? When testing, I got the error 
```
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Stateless function components cannot be given refs (See ref "test7" in a component created by Checkbox). Attempts to access this ref will fail.
```

There aren't any calls to the ref in the file.

**Comment by JakeDawkins** [comment]
Also, Don't merge this :) Found a rather nasty problem.

**Comment by johnthepink** [comment]
@JakeDawkins yea I'm not sure, but looks like it won't work now anyways :)

**Comment by JakeDawkins** [comment]
This should work now @johnthepink

File: imports/components/forms/Checkbox.js [hunk 1]
```diff
@@ -81,12 +74,11 @@ export default class Checkbox extends React.Component {
         </h6>
 
         <input
-            ref={this.props.id || this.props.label || this.props.name}
             id={this.props.id || this.props.label || this.props.name}
             type={this.props.type || "checkbox"}
             name={this.props.name || this.props.label}
             className={this.props.inputClasses}
-            disabled={this.disabled()}
+            disabled={this.props.disabled ? true : false}
```
Comment (johnthepink) [inline imports/components/forms/Checkbox.js]: I like this better than the function. Any reason why this couldn't just be `this.props.disabled`?

Comment (johnthepink) [inline imports/components/forms/Checkbox.js]: Oh i guess this handles the case of `this.props.disabled` being undefined. Carry on. I still think we should have prop types though

**Comment by JakeDawkins** [comment]
@johnthepink I think I got all the props. I changed the "inputClasses" prop to an array instead of a string since that's already how we pass classes to the component. Thoughts?

**Comment by johnthepink** [comment]
@JakeDawkins I like it

**Pull request merged**